benchmark/runners/with-cloud-history.ts

```diff
@@ -6,18 +6,23 @@
  * ./run benchmark/runners/with-cloud-history.ts --bundle
  * ```
  */
+
 import { initializeBindings } from 'o1js';
 import { logResult } from '../benchmark.js';
 import { EcdsaBenchmark } from '../benchmarks/ecdsa.js';
+import { InitBenchmark } from '../benchmarks/init.js';
```

Comment:
I think this doesn't work, because we're already importing o1js here before trying to measure o1js import time. Node.js will be smart enough to not compile and run the module a second time, so we're not getting the proper benchmark

Reply:
also `EcdsaBenchmark` imports o1js